lib/Activity/ApprovalProvider.php

```diff
@@ -116,9 +116,10 @@ public function parse($language, IEvent $event, ?IEvent $previousEvent = null):
 
 			$file = [
 				'type' => 'file',
-				'id' => $event->getObjectId(),
+				'id' => (string)$event->getObjectId(),
 				'name' => $event->getObjectName(),
-				'path' => $path,
+				'path' => trim($path, '/'),
```

Comment:
Why trim the slash from the beginning?

Reply:
Because they do so there: https://github.com/nextcloud/server/blob/master/apps/files_sharing/lib/Activity/Providers/Base.php#L109
but no strong reason.